### DISCUSSION TITLE
Requesting test funds

### DISCUSSION THREAD

**Discussion by Denchous** [entropyxyz/community#116] [Get Test Funds] [answered]
### Address

5FA9Nif4KVo7ujhaae1bJPDjHLJA6hoV4s8atsPdtHpoowrf

**Comment by johnnymatthews** [comment, marked as answer]
Sent!

**Comment by johnnymatthews** [comment]
Just sent you some more funds @Denchous. Go ahead and [check your balance](https://docs.entropy.xyz/reference/cli#balance) again.
